src/Raven.Server/Smuggler/Documents/StreamSource.cs

```diff
@@ -680,7 +680,19 @@ public async Task<DatabaseRecord> GetDatabaseRecordAsync()
                             _log.Info("Wasn't able to import the IndexesHistory from smuggler file. Skipping.", e);
                     }
                 }
-                
+
+                if (reader.TryGet(nameof(databaseRecord.DatabaseVersion), out long? databaseVersion) && databaseVersion != null)
+                {
+                    try
+                    {
+                        databaseRecord.DatabaseVersion = databaseVersion;
+                    }
+                    catch (Exception e)
+                    {
```

Comment:
can this even fail?

Reply:
nope :) removed

Reply:
was this removed?

Reply:
@shiranshalom ?

Reply:
I'm not sure why this still exists. it was removed a while ago, and if you look at the entire file, it’s not there